tests/integration/CMakeLists.txt

```diff
@@ -15,9 +15,10 @@ if(MOCOCRW_HSM_ENABLED)
     add_executable(hsm-integration-test hsm-integration-test.cpp)
     target_link_libraries(hsm-integration-test PRIVATE MoCOCrW::mococrw)
     target_compile_definitions(hsm-integration-test PRIVATE HSM_ENABLED)
+    # libp11 writes to stderr on unsuccessful key loading. Pipe it to devnull
```

Comment:
Isn't this bad for debugging?

Reply:
Can you write it to a file?
Then we can at least check the content of the file on local execution. Maybe there is also a way to make this file available after test execution in github actions.